### DISCUSSION TITLE
不懂就问：安装拉取地址是飞致云域名，而不是gihub的仓库

### DISCUSSION THREAD

**Discussion by SnailBird** [1Panel-dev/1Panel#5965] [General]
不懂就问：
1、安装1panel时，curl命令的地址是飞致云的域名，而不是GitHub的代码仓库？如何保证安装的代码和GitHub上的一致呢？
2、GitHub上Releases的最新版本，如何直接下载到Linux主机上进行安装？

**Comment by wanghe-fit2cloud** [comment]
针对你的疑问，建议你可以考虑根据源码自行构建，官方文档中提供了开发环境搭建手册。

**Reply by SnailBird** [reply]
您好，方便发一下官方文档的安装地址吗？

**Reply by wanghe-fit2cloud** [reply]
参考：https://1panel.cn/docs/dev_manual/dev_manual/
